packages/yourloops/components/device/parameters-change-history.tsx

```diff
@@ -59,17 +59,18 @@ interface ParametersChangeHistoryProps {
 export const ParametersChangeHistory: FC<ParametersChangeHistoryProps> = ({ history, goToDailySpecificDate }) => {
   const theme = useTheme()
   const { t } = useTranslation()
+  const [historyToDisplay, setHistoryToDisplay] = useState<ParametersChange[]>([])
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏As suggested earlier, could be renamed to `sortedHistory`